generated/proto/commonpb/common.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Sure thing, in that case you can also name the directory "common" and have the package name be "commonpb", i don't feel strongly about it though